package in.co.exception;

public class FinallyWithoutException {
	public static void main(String[] args) {
		try {
			int i = 23;
			System.out.println(i);
	}
		    finally  {
		    	System.out.println("Stylish");
		    }
	}

}
